frontend/packages/git-service/src/services/bitbucket-service.ts

```diff
@@ -125,13 +127,17 @@ export class BitbucketService extends BaseService {
     }
   };
 
-  isDockerfilePresent = () => this.isFilePresent(`${this.metadata.contextDir}/Dockerfile`);
+  isDockerfilePresent = () =>
+    this.isFilePresent(`${this.metadata.contextDir}/${this.metadata.dockerfilePath}`);
 
-  getDockerfileContent = () => this.getFileContent(`${this.metadata.contextDir}/Dockerfile`);
+  getDockerfileContent = () =>
+    this.getFileContent(`${this.metadata.contextDir}/${this.metadata.dockerfilePath}`);
 
-  isDevfilePresent = () => this.isFilePresent(`${this.metadata.contextDir}/devfile.yaml`);
+  isDevfilePresent = () =>
+    this.isFilePresent(`${this.metadata.contextDir}${this.metadata.devfilePath}`);
 
-  getDevfileContent = () => this.getFileContent(`${this.metadata.contextDir}/devfile.yaml`);
+  getDevfileContent = () =>
+    this.getFileContent(`${this.metadata.contextDir}${this.metadata.devfilePath}`);
 
```

Comment:
These functions can also be combined into one if we use generic `filePath` property.